classes/Settings/FormConfiguration/class.xudfFormConfigurationGUI.php

```diff
@@ -144,22 +174,30 @@ protected function update(): void
 
     protected function edit(): void
     {
-        $element = xudfContentElement::find($_GET['element_id']);
+        $elementId = $this->httpWrapper->query()->retrieve(
+            xudfFormConfigurationFormGUI::F_ELEMENT_ID,
+            $this->refinery->kindlyTo()->int()
```

Comment:
s.a.

Reply:
Using that throws an exception since ids in the query are not of type integer,

kindlyTo turns strings into integer.
to expects it to already be an integer